superbench/benchmarks/registry.py

```diff
@@ -105,32 +105,19 @@ def __get_benchmark_name(cls, benchmark_context):
         return benchmark_name
 
     @classmethod
-    def check_parameters(cls, benchmark_context):
-        """Check the validation of customized parameters.
+    def create_benchmark_context(cls, name, platform=Platform.CUDA, parameters='', framework=Framework.NONE):
+        """Constructor.
 
         Args:
-            benchmark_context (BenchmarkContext): the benchmark context.
+            name (str): name of benchmark in config file.
+            platform (Platform): Platform types like CUDA, ROCM.
```

Comment:
Shall we change to `Platform.CPU, Platform.CUDA, Platform.ROCM`?

Reply:
OK